kubernetes-manifests/frontend.yaml

```diff
@@ -51,21 +45,6 @@ spec:
         envFrom:
         - configMapRef:
             name: environment-config
-        - configMapRef:
-            name: service-api-config
-        readinessProbe:
-          httpGet:
-            path: /ready
-            port: 8080
-          initialDelaySeconds: 5
-          periodSeconds: 5
-      volumes:
-      - name: publickey
-        secret:
-          secretName: jwt-key
-          items:
-          - key: jwtRS256.key.pub
-            path: publickey
```

Comment:
is this a permanent change or just for the boilerplate deployment?

Reply:
The readiness probe is removed just for the boilerplate deployment. 

The `jwt-key` will no longer be needed since we're just checking for the existence of the token before displaying the homepage and letting the backend services do their own JWT validation.

Lastly, the `service-api-config` configmap won't be used since we'll be using Dan's suggestion of creating a k8s Ingress at `/api` to access backend services while the frontend will sit at `/`. This allows us to use relative paths when we make requests from the frontend (e.g `http.get('api/login')`. WDYT?

Reply:
thanks for explaining this, SGTM. 
`service-api-config`: contains backend addresses, removed for Ingress change
`jwt-key`: removing token validation from frontend, will depend on backend to validate

I wonder if keeping the `readinessProbe` on the `/` path could be helpful, so we don't forget to add it back in later.